src/rasterstats/main.py

```diff
@@ -34,7 +36,13 @@ def zonal_stats(*args, **kwargs):
 
     The only difference is that ``zonal_stats`` will
     return a list rather than a generator."""
-    return list(gen_zonal_stats(*args, **kwargs))
+    progress = kwargs.get("progress")
```

Comment:
Consider raising an exception if progress is true but tqdm is None so we can give the user a more accurate error message in this case. Otherwise we'd get `TypeError: 'NoneType' object is not callable`